gitops_cli_argocd/argocd-gitops-cli-reference.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="argocd-gitops-cli-reference"]
+= {gitops-shortname} argocd CLI reference
+:context: argocd-gitops-cli-reference
+
+toc::[]
+
+:FeatureName: The {gitops-title} `argocd` CLI tool
+include::snippets/technology-preview.adoc[]
+
+This section lists the basic {gitops-shortname} `argocd` CLI commands.
+
+[NOTE]
+====
+When you install the {gitops-shortname} `argocd` CLI for MicroShift, it does not host an Argo CD server. Instead, it only supports command execution in the  `core` mode. In this mode, you can use the `--core` argument to communicate directly with the Kubernetes API server based on the settings in the  `KUBECONFIG` environment variable or the default `$HOME/.kube/config` file. Therefore, you don't need to log in to the Argo CD server to execute commands.
```

Comment:
Probably we can  remove this note. Let's put this note in MicroShift specfic Documentation. This paragraph can cause confusion as it might give an impression that there is a CLI for MicroShift that needs to be installed.